eng/Versions.props

```diff
@@ -70,12 +73,12 @@
     <AspNetCorePackageVersionFor31Templates>$(AspNetCoreVersion)</AspNetCorePackageVersionFor31Templates>
     <!-- 3.0 Template versions -->
     <MicrosoftDotnetWinFormsProjectTemplates30PackageVersion>4.8.0-rc2.19462.10</MicrosoftDotnetWinFormsProjectTemplates30PackageVersion>
-    <MicrosoftDotNetWpfProjectTemplates30PackageVersion>3.0.0-rc2.19459.11</MicrosoftDotNetWpfProjectTemplates30PackageVersion>
+    <MicrosoftDotNetWpfProjectTemplates30PackageVersion>3.0.0</MicrosoftDotNetWpfProjectTemplates30PackageVersion>
     <NUnit3Templates30PackageVersion>1.6.3</NUnit3Templates30PackageVersion>
     <MicrosoftDotNetCommonItemTemplates30PackageVersion>2.0.0-preview8.19373.1</MicrosoftDotNetCommonItemTemplates30PackageVersion>
```

Comment:
This version is what is in release/3.0.1xx, not flowed by darc and looks wrong. Will investigate separately.

Reply:
This is not actually resolved, but is not part of this change. Will tackle next.